storage/txcache/txCache.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 var _ storage.Cacher = (*TxCache)(nil)
+var _ txCache = (*TxCache)(nil)
 
```

Comment:
Put and HasOrAdd have some unused parameters.

Reply:
Fixed / `_` (these methods of the `Cacher` interface are not implemented by the `TxCache`).